src/components/ApplicationMenu/ApplicationMenu.js

```diff
@@ -177,8 +177,8 @@ class ApplicationMenu extends Component<Props> {
       });
 
       template.splice(editMenuIndex, 0, {
-        id: 'project',
-        label: isMac ? 'Project' : '&Project',
+        id: 'current-project',
+        label: isMac ? 'Current Project' : 'Current &project',
```

Comment:
Curious what y'all think about this. I find it way nicer, since it's clear that "Project" is specific to the current selection.

Reply:
Also, Windows folks (@superhawk610, @AWolf81), I assume the ampersand `&` is to select which key can be used for the menu item? I think `p` makes the most sense here?

Reply:
Ah, so that's what the ampersand does. Was wondering about that.

Reply:
Yeah, took me a minute to figure out as well 🙃

Unless I'm doing something wrong, however, those shortcuts (or accelerators, as I think the official docs calls them) are universal among all sub menus, so `&Foo` in the **File** menu would conflict with `&Fuz` in the **Edit** menu. They also don't necessarily correlate to their respective `CmdOrCtrl` shortcuts, but instead to `Alt` shortcuts...? So `&Foo` would be hotkeyed to `Alt+F`.

To further complicate things, you can't reuse any letters that your menu titles begin with, as they're automatically assigned to a hotkey by Electron. So, for example, `File > &Bar` would be `Alt+F+B`, but `File > &Foo` just wouldn't do anything, since it requires a third key and `Alt+F+F` doesn't seem to work for that.

Not really sure we need them at all, just a convention I've seen other Electron apps use.